utils/build-script-impl

```diff
@@ -2013,6 +2054,21 @@ for deployment_target in "${STDLIB_DEPLOYMENT_TARGETS[@]}"; do
                 echo "--- Finished tests for ${product} ---"
                 continue
                 ;;
+            libdispatch)
+                if [[ "${SKIP_TEST_LIBDISPATCH}" ]]; then
+                    continue
+                fi
+
+                LIBDISPATCH_BUILD_DIR=$(build_directory ${deployment_target} ${product})
```

Comment:
Would you mind printing a "Running tests" banner like other cases do?  This makes CI output much better.